src/LogWindow.h

```diff
@@ -0,0 +1,22 @@
+/**********************************************************************
+
+Audacity: A Digital Audio Editor
+
+LogWindow.h
+
+Paul Licameli split from AudacityLogger.h
+
+**********************************************************************/
+
+#ifndef __AUDACITY_LOG_WINDOW__
+#define __AUDACITY_LOG_WINDOW__
+
+//! Maintains the unique logging window which displays debug information
+class LogWindow
+{
+public:
+   //! Show or hide the unique logging window; create it on demand the first time it is shown
+   static void Show(bool show = true);
```

Comment:
`show` parameter used for show (`show` = `true`) or hide (`show` = `false`)?
Why you don't using separate methods for this or instead bool variable using `enum name { kShowDialog, kHideDialog };`?

Reply:
How I understand `bool` parameter for show/hide its standard wxWidgets conception.